mesh/mesh_readers.cpp

```diff
@@ -3037,9 +3037,9 @@ static void ReadCubitDimensions(const int netcdf_descriptor,
 
 static void ReadCubitBoundaries(const int netcdf_descriptor,
                                 const int num_boundaries,
-                                std::vector<size_t> &num_boundary_elements,
-                                int **boundary_elements,
-                                int **boundary_sides)
+                                vector<size_t> &num_boundary_elements,
+                                vector<vector<int>> boundary_elements,
+                                vector<vector<int>> boundary_sides)
```

Comment:
Is it necessary to pass by value here? Maybe pass by reference to avoid copies; if passing by reference is fine, also make the argument const -- it seems to be an input-only parameter.

Reply:
Good catch, this was a bug. They should definitely be passed by reference (but not const, since they are modified in the function).

Reply:
Fixed in 43c7303e4